core/src/main/java/io/questdb/griffin/SqlOptimiser.java

```diff
@@ -4557,31 +4560,43 @@ private QueryModel rewriteSampleBy(@Nullable QueryModel model) throws SqlExcepti
         QueryModel nested = model.getNestedModel();
```

Comment:
nit: javadoc for `rewriteSampleBy` still mentions
```
Therefore, the rewrite avoids the following:
     - fills
```
which is no longer correct. The javadoc should be updated.